tools/pd-ctl/pdctl/command/gc_safepoint_command.go

```diff
@@ -41,47 +35,36 @@ func NewServiceGCSafepointCommand() *cobra.Command {
 // NewDeleteServiceGCSafepointCommand return a subcommand to delete service gc safepoint
 func NewDeleteServiceGCSafepointCommand() *cobra.Command {
 	l := &cobra.Command{
-		Use:    "delete <service ID>",
-		Short:  "delete a service gc safepoint",
-		Run:    deleteSSP,
-		Hidden: true,
+		Use:               "delete <service ID>",
+		Short:             "delete a service gc safepoint",
+		PersistentPreRunE: requirePDClient,
```

Comment:
This is unnecessary since it's a sub-command of `service-gc-safepoint` which will inherit the same `PersistentPreRunE`.